packages/modular-scripts/react-scripts/config/webpack.config.js

```diff
@@ -102,8 +102,18 @@ module.exports = function (webpackEnv) {
   const env = getClientEnvironment(paths.publicUrlOrPath.slice(0, -1));
 
   // common function to get style loaders
-  const getStyleLoaders = (cssOptions, preProcessor) => {
+  const getStyleLoaders = (cssOptions, preProcessor, includeEsmLoader) => {
     const loaders = [
+      // This loader translates external css dependencies if we're using a CDN
+      // Since it's a pitching loader, it's important that it stays at the top
+      // excluding all the others in the chain if it's triggered
+      includeEsmLoader &&
+        function externalStyleLoader(info) {
```

Comment:
There's already an import for `info` on line 17 of this file. I know this new variable is scoped to this function, but I wonder whether its name would be clearer if it were more specific.

Reply:
We decided to not destructure in line 17 (`const { info } = require('../../react-dev-utils/logger');`)

Reply:
Done

Reply:
Done